app/src/main/java/com/example/parkingsystem_kotlin/mvp/presenter/ParkingSizePresenter.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.parkingsystem_kotlin.mvp.presenter
+
+import com.example.parkingsystem_kotlin.mvp.contract.ParkingSizeContract
+
+class ParkingSizePresenter(private val model: ParkingSizeContract.Model, private val view: ParkingSizeContract.View): ParkingSizeContract.Presenter {
+
+    override fun onParkingSizeButtonPressed() {
+        val parkingSize = model.getParkingSize()
+        view.showParkingSize(parkingSize)
```

Comment:
you can do here

view.showParkingSize(model.getParkingSize())

Reply:
done